_posts/2019-04-09-sr2019-tlc-crowned-champion.md

```diff
@@ -79,7 +79,7 @@ We love to see teams dressing up to compliment their robot theme, and this year
 
 Lastly, we like to see the robot-building process over the course of the year, and Collyer's were keeping us up-to-date on [Twitter](https://twitter.com/CollyersRobots) all the way from Kickstart.
 
-For full details on all the awards, please see the [rulebook](/docs/resources/2019/rulebook.pdf).
+For full details on all the awards, please see the [rulebook](https://studentrobotics.org/docs/docs/resources/2019/rulebook.pdf).
```

Comment:
Think there's an extra layer of `/docs` here

Reply:
not sure how that happened. I checked all the new links too!
eb1a36860a377ea49d4dd70757ef8a416cc76b4e fixes